packages/mobile/src/screens/profile-screen/ArtistRecommendations/ArtistRecommendations.tsx

```diff
@@ -102,12 +94,12 @@ export const ArtistRecommendations = (props: ArtistRecommendationsProps) => {
   const artistsToFollow = useSelector<CommonState, User[]>((state) =>
     Object.values(getUsers(state, { ids: idsToFollow || [] }))
   )
-  const suggestedArtists = useSelector((state) =>
-    selectSuggestedFollowsUsers(state, { id: user_id })
-  )
+
+  const { data: suggestedArtists } = useRelatedArtists(user_id)
```

Comment:
do we need to filter out users with `current_user_follows`?
I think that was part of the `suggestedFollows` logic